src/Fixtures/en/frisbeeDrills.en.js

```diff
@@ -5281,7 +5286,8 @@ const kyeDrill = {
   author: 'Kye',
   title: '3 vs 2',
```

Comment:
The drill is called 3v2 but on the animation it's 4v2 ? :thinking:

Reply:
Je crois que c'est juste 3 autres joueurs en attente superposés